_scicomputing/store_overview.md

```diff
@@ -6,7 +6,7 @@ primary_reviewers:
 
 The Hutch provides many platforms for storing, using, moving, and sharing your data- relational databases, block-based network file storage, and cloud object datastores.  Each of these platforms have options as well- these options to address the diverse needs of modern biomedical and bioinformatic research.  The platform you use will depend on the nature of the data and how you plan to use it.
 
-This article is an overview of the available storage platforms and options and provides some guidance for selecting the option best for your need. More detailed information- including setup instructions and limitations- can be found in the pages and articles linked in the sidebar.  There are multiple resources at the Fred Hutch to help you design and implement a data management plan for your research.  The Fred Hutch [Data Science Lab](https://hutchdatascience.org/) offers a [Data House Calls](https://hutchdatascience.org/datahousecalls/) service to get a consultation to discuss your data needs and how you might design a plan, and Scientific Computing can help you with the tools and resources you need to implement that plan (email `scicomp` for help).
+This article is an overview of the available storage platforms and options and provides some guidance for selecting the option best for your need. More detailed information- including setup instructions and limitations- can be found in the pages and articles linked in the sidebar.  There are multiple resources at the Fred Hutch to help you design and implement a data management plan for your research.  The Fred Hutch [Data Science Lab](https://hutchdatascience.org/) offers a [Data House Calls](https://calendly.com/data-house-calls/management?back=1&month=2024-10) service to get a consultation to discuss your data needs and how you might design a plan, and Scientific Computing can help you with the tools and resources you need to implement that plan (email `scicomp` for help).
```

Comment:
```suggestion
This article is an overview of the available storage platforms and options and provides some guidance for selecting the option best for your need. More detailed information- including setup instructions and limitations- can be found in the pages and articles linked in the sidebar.  There are multiple resources at the Fred Hutch to help you design and implement a data management plan for your research.  The Fred Hutch [Data Science Lab](https://hutchdatascience.org/) offers a [Data House Calls](https://calendly.com/data-house-calls/management) service to get a consultation to discuss your data needs and how you might design a plan, and Scientific Computing can help you with the tools and resources you need to implement that plan (email `scicomp` for help).
```

Reply:
Great call on making the link specific to the Data Management DHC! Just wanted to remove the month specification that gets added automatically by Calendly. The link probably still works with that, but might be smart just in case.